### DISCUSSION TITLE
Get Post Flair

### DISCUSSION THREAD

**Discussion by cachho** [thislooksfun/snoots#119] [Q&A] [answered]
Hi, cool project. Maybe I'm missing something, I just got started with this today. I'm on `1.0.0-dev.28 `

I'm missing a "post flair" property on the Post class. 

Is this called something else? Or am I looking in the wrong place?

**Comment by thislooksfun** [comment, marked as answer]
You're not missing anything, post flairs have not been implemented yet, as I have not had the time (and motivation) to properly get them set up.

https://github.com/thislooksfun/snoots/blob/ab02e19347d000dcf5b5fa03164c5e1e295ec14e/src/reddit/post/object.ts#L147-L154

**Reply by cachho** [reply]
should I hold my breath?

**Reply by thislooksfun** [reply]
I'll try to get to this in the next few days, but since this is just a hobby project of mine I can't make any promises.

**Reply by cachho** [reply]
awesome thanks. I was actually gonna see if I can contribute. Next thing that I'm dearly missing would be **post thumbnails**.

**Reply by thislooksfun** [reply]
It's looking like I will be able to work on flairs this weekend, but contributions are always welcome!

**Reply by thislooksfun** [reply]
> I will be able to work on flairs this weekend

Haha! Wrong! Instead I shall be sick and miserable. 🤒

**Reply by cachho** [reply]
alright. Did you see my PR for thumbnails?

**Reply by cachho** [reply]
and get well soon 🍀

**Reply by thislooksfun** [reply]
I see all that happens here 👀. I just haven't had a change to review it yet.
